packages/regl-worldview/src/commands/Command.js

```diff
@@ -83,10 +88,15 @@ export default class Command<T> extends React.Component<Props<T>> {
   }
 
   _getHitmapPropFromHitmapId(objectId: number) {
-    const { hitmapProps = [] } = this.props;
+    const { hitmapProps = [], testObjectIdInRange } = this.props;
+
     return hitmapProps.find((hitmapProp) => {
-      // TODO handle objects with 'colors' property
-      if (hitmapProp.color) {
+      if (hitmapProp.points) {
```

Comment:
I don't think we can rely on an object having `.points` as it being an instanced object - some things may have `.points` and just be a single marker.  We might need to rethink how we handle custom hitmap props a bit for custom `<Command />`s.